src/proxy-ui-api/src/test/java/org/niis/xroad/restapi/openapi/OpenapiApiControllerTest.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * The MIT License
+ *
+ * Copyright (c) 2019- Nordic Institute for Interoperability Solutions (NIIS)
+ * Copyright (c) 2018 Estonian Information System Authority (RIA),
+ * Nordic Institute for Interoperability Solutions (NIIS), Population Register Centre (VRK)
+ * Copyright (c) 2015-2017 Estonian Information System Authority (RIA), Population Register Centre (VRK)
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.niis.xroad.restapi.openapi;
+
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.core.io.Resource;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.test.context.support.WithMockUser;
+
+import java.io.IOException;
+
+import static org.junit.Assert.assertEquals;
+import static org.niis.xroad.restapi.openapi.OpenapiApiController.OPENAPI_DEFINITION_FILENAME;
+
+public class OpenapiApiControllerTest extends AbstractApiControllerTestContext {
+
+    @Autowired
+    OpenapiApiController openapiApiController;
+
+    @Test
+    @WithMockUser(authorities = { "DOWNLOAD_OPENAPI" })
```

Comment:
This line should probably be removed, since permission DOWNLOAD_OPENAPI was removed? 

MockUser is not needed, and probably better off, since having one in the test can even be a tad misleading.

Reply:
Removed the extra line in commit https://github.com/nordic-institute/X-Road/pull/930/commits/66609ecb6c6e8886234224793ebd454300dc5ee3